cmis/src/main/java/org/frankframework/extensions/cmis/CmisSender.java

```diff
@@ -1143,6 +1144,7 @@ public String getTrustManagerAlgorithm() {
 		return sessionBuilder.getTrustManagerAlgorithm();
 	}
 
+	@Unsafe(description = "verifyHostname is unsafe and should not be used in a production environment.")
```

Comment:
The text that in the annotation is computable. Perhaps it's better to have an empty annotation and generate the text.

Reply:
I choose for a description, because there could be use cases where it is nice to have a different text, but I guess for now it's not really necessary.